modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties

```diff
@@ -5,6 +5,7 @@ default-search-result-permission-filter-configuration-name=Default Search Result
 disabled-entry-class-names=Disabled Entry Class Names
 disabled-entry-class-names-help=Specify the entry class names for which DefaultKeywordQueryContributor will not include the description, userName, and title fields as part of the keyword search query.
 exact-match-boost=Exact Match Boost
+exact-match-boost-help=The boost value applied when search terms match a title as an exact phrase.
```

Comment:
We should use full sentences for our help keys. I know many of the existing keys here aren't full sentences, but I have to get in here and do a review, and change that.
How about this?
> Apply a boost value for hits with titles exactly matching the searched keyword or phrase.

Reply:
is "Set the boost value to be applied when search terms match a title as an exact phrase." a full sentence?

Reply:
Yep.